### ISSUE TITLE
ENet-Label-Torch: how to modify the pre-trained model for training data with different resolution

### ISSUE THREAD

**Issue by zoeveryday** [cardwing/Codes-for-Lane-Detection#122]
In ENet-Label-Torch project, is the pre-trained model (the one provided in training - step 1) generated from /experiments/models/SCNN-gen.lua?

According to the instructions here https://github.com/XingangPan/SCNN/issues/7#issuecomment-358311752, if we want to modify the pre-trained model for different training date, the third step is to change the Num, length and re-generate the model from vgg. So, can we still follow this step?

Thanks!

**Comment by cardwing** [comment]
The pre-trained model is just ENet, not SCNN. If you want to use images with different resolutions, you just need to modify the nn.view() and nn.linear() in the lane existence prediction branch.

**Comment by zoeveryday** [comment]
@cardwing got it, thanks!
